packages/ckeditor5-show-blocks/README.md

```diff
@@ -0,0 +1,21 @@
+CKEditor 5 show blocks feature
+================================
+
+[![npm version](https://badge.fury.io/js/%40ckeditor%2Fckeditor5-show-blocks.svg)](https://www.npmjs.com/package/@ckeditor/ckeditor5-show-blocks)
+[![Coverage Status](https://coveralls.io/repos/github/ckeditor/ckeditor5/badge.svg?branch=master)](https://coveralls.io/github/ckeditor/ckeditor5?branch=master)
+[![Build Status](https://travis-ci.com/ckeditor/ckeditor5.svg?branch=master)](https://app.travis-ci.com/github/ckeditor/ckeditor5)
+![Dependency Status](https://img.shields.io/librariesio/release/npm/@ckeditor/ckeditor5-show-blocks)
+
+This package contains the show blocks feature for CKEditor 5. It allows you to display the HTML element names for content blocks.
```

Comment:
I wonder if a description from https://ckeditor.com/cke4/addon/showblocks makes more sense. WDYT @godai78?

Reply:
We could combine these. Surely the "show block feature" says very little about what this really is, so some kind of explanation is very much in order.